cloudscale/resource_cloudscale_server.go

```diff
@@ -56,6 +56,7 @@ func getServerSchema() map[string]*schema.Schema {
 			Type:     schema.TypeInt,
 			Optional: true,
 			ForceNew: true,
+			Default:  10,
```

Comment:
I'm not sure if this necessary anymore, since we kind of discussed this?

Reply:
Agree, was just playing around with terraform, and checking if it actually broke something. I will remove it.